plugins/duck.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: duck
+spec:
+  version: v0.0.1
+  homepage: https://github.com/n3wscott/kubectl-duck
+  shortDescription: Ducktype support for kubectl.
```

Comment:
Can you be more descriptive? Since it's a kubectl, it's already `for kubectl`, that part is redundant. I would probably say something like:

> List custom resources with ducktype support

Reply:
OK, I will use this for now but I will be updating to also support things like podspecable which are things like Deployments and Services.sources.knative.dev in the next release, will update this then.